File: Assignment 3/Assignment 3/problem2.java
//accept no from user and print event factors of that number.

import java.lang.*;
import java.util.*;

class EvenFactor
{
	public static void EvnFact(int iNo)
	{
		int iCnt = 0;
		
		for(iCnt=1;iCnt<=iNo;iCnt++)
		{
			if((iNo%iCnt==0)&&(iCnt%2==0))
			   {
				   System.out.println(iCnt);  
			   }
		}
	}	
}

class problem2
{
	public static void main(String a[])
	{
		EvenFactor eobj = new EvenFactor();
		 
		Scanner sobj =new Scanner(System.in);
		
		System.out.println("Ener Number");
		int iValue = sobj.nextInt();
		
		eobj.EvnFact(iValue);
	}
}

File: Assignment 12/Problem2.c
// Accept number from user and Display below pattern
// input =3;
// output = 3 # 2 # 1 #

#include<stdio.h>

void Display(int iNo)
{
	int iCnt=0;
	
	for(iCnt=iNo;iCnt>=1;iCnt--)
	{
		printf("%d\t#\t",iCnt);
	}
}
int main()
{
	int iValue=0;
	
	printf("Enter Number of Row\n");
	scanf("%d",&iValue);
	
	Display(iValue);
	
	return 0;
}